### ISSUE TITLE
RabbitMQNodeDown alert is always active

### ISSUE THREAD

**Issue by fitz123** [grafana/jsonnet-libs#944]
Looks like the RabbitMQNodeDown [expr](https://github.com/grafana/jsonnet-libs/blob/master/rabbitmq-mixin/alerts/clusterAlerts.yaml#L41) is always evaluated to true if you have at least 1 cluster running. It should probably be changed to 
```
-    expr: sum by (rabbitmq_cluster) (rabbitmq_identity_info)
+    expr: sum by (rabbitmq_cluster) (rabbitmq_identity_info) < 3
```

**Comment by Irmius** [comment]
Facing the same issue. 
It would be nice if it would be fixed

**Comment by v-zhuravlev** [comment]
@Irmius , this should be fixed #945
